README.md

```diff
@@ -31,7 +31,7 @@ The following table captures the compatibility matrix of the core filestore driv
 | master                                  | no   |  yes |  yes |  yes  |
 
 The manifest bundle which captures all the driver components (driver pod which includes the containers csi-external-provisioner, csi-external-resizer, csi-external-snapshotter, gcp-filestore-driver, csi-driver-registrar;
-csi driver object, rbacs, pod security policies etc) can be picked up from the master branch [overlays](deploy/kubernetes/overlays) directory. We structure the overlays directory, per minor version of kubernetes because not all driver components can be used with all kubernetes versions. For example volume snapshots are supported 1.17+ kubernetes versions thus [stable-1-16](deploy/kubernetes/overlays/stable-1-16) driver manifests does not contain the snapshotter sidecar.
```

Comment:
nit: change the semicolon after csi-driver-registrar to a comma? And the comman in following sentence is a little weird, maybe to drop it: "We structure the overlays directory per minor version of kubernetes because not all driver components..."

Reply:
done